sources/web/datalab/static/datalab.js

```diff
@@ -282,10 +282,10 @@ function initializeNotebookApplication(ipy, notebook, events, dialog, utils) {
       return lastSepPos >= 23 &&
           path.substring(lastSepPos-23, lastSepPos) == '/datalab/docs/notebooks';
     }
-    
+
```

Comment:
FYI: I'm fine with the whitespace changes for this review, but make sure your editor is configured to preserve whitespace on save, so that pull requests only include the relevant changes.